storagenode/storagenodedb/database.go

```diff
@@ -225,17 +273,11 @@ func (db *DB) closeDatabases() error {
 
 // closeDatabase closes the specified SQLite database connections and removes them from the associated maps.
 func (db *DB) closeDatabase(dbName string) (err error) {
-	conn, ok := db.sqlDatabases[dbName]
+	mdb, ok := db.sqlDatabases[dbName]
 	if !ok {
-		return ErrDatabase.New("double close on database %s", dbName)
+		return ErrDatabase.New("no database with name %s", dbName)
 	}
-	delete(db.sqlDatabases, dbName)
```

Comment:
Wait, why was the delete removed?

Reply:
> Wait, why was the delete removed?

This looks odd if you only look at this diff line but the business objects added to the map never get replaced. We reconfigure them on DB open/close but we never have to remove them. We add them all during process start up.

This type of wasteful churn sometimes happens when you split PR's and do some infrastructure refactoring for future use PR and then you finish the feature implementation PR and find out you need something slightly different.